osh/word_.py

```diff
@@ -620,6 +620,32 @@ def LiteralToken(UP_w):
   return None
 
 
+def WordAsKeywordToken(UP_w):
+  # type: (word_t) -> Token
+  """
+  Given a word that IS A keyword, return the single token at the start.
+
+  In C++, this casts without checking, so BE CAREFUL to call it in the right context.
+  """
+  assert UP_w.tag() == word_e.Compound, UP_w
+  w = cast(CompoundWord, UP_w)
+
+  part = w.parts[0]
+  assert part.tag() == word_part_e.Literal, part
+  return cast(Token, part)
```

Comment:
I wonder if we should `assert consts.GetKind(tok.id) == Kind.KW` here?   I don't think there's any case where that's false?

That is sort of what the comment is saying -- "given that the word is a keyword"

----

Also how about shortening the names like

```
word_.AsKeywordToken()
word_.AsOperatorToken()
```

?

Since we always import and use `module.Func()`, I think this shortens the code without affecting readability

Reply:
> I wonder if we should assert consts.GetKind(tok.id) == Kind.KW here? I don't think there's any case where that's false?

I'll check to make sure that this assumption actually is correct. If so, I'll add it.

Re: changing the names. That seems reasonable, especially if we adhere to the `module.Func()` pattern everywhere.